Keep awscrt pinned. This is the only user, and presumably
Amazon has a good reason for not bumping the version yet.

N.B. unpinned awscrt to fix #66141.

--- aws-cli-2.9.6-orig/pyproject.toml	2022-12-11 17:14:55.655896748 -0500
+++ aws-cli-2.9.6/pyproject.toml	2022-12-11 17:16:25.306058469 -0500
@@ -1,6 +1,6 @@
 [build-system]
 requires = [
-"flit_core>=3.7.1,<3.7.2",
+"flit_core",
 ]
 build-backend = "pep517"
 backend-path = ["backends"]
@@ -27,16 +27,16 @@
     "Programming Language :: Python :: 3.10",
 ]
 dependencies = [
-    "colorama>=0.2.5,<0.4.4",
-    "docutils>=0.10,<0.20",
-    "cryptography>=3.3.2,<=38.0.1",
-    "ruamel.yaml>=0.15.0,<=0.17.21",
-    "prompt-toolkit>=3.0.24,<3.0.29",
-    "distro>=1.5.0,<1.6.0",
-    "awscrt>=0.12.4,<=0.14.0",
-    "python-dateutil>=2.1,<3.0.0",
-    "jmespath>=0.7.1,<1.1.0",
-    "urllib3>=1.25.4,<1.27",
+    "colorama",
+    "docutils",
+    "cryptography",
+    "ruamel.yaml",
+    "prompt-toolkit",
+    "distro",
+    "awscrt",
+    "python-dateutil",
+    "jmespath",
+    "urllib3",
 ]
 dynamic = ["version"]

